app/code/Magento/Cms/Controller/Adminhtml/Page/Delete.php

```diff
@@ -30,7 +49,7 @@ public function execute()
             $title = "";
             try {
                 // init model and delete
-                $model = $this->_objectManager->create(\Magento\Cms\Model\Page::class);
+                $model = $this->_cmsPage;
```

Comment:
Could we not directly use the `$this->cmsPage` here instead of setting against the `$model` variable.